tools/cldr-apps/src/main/webapp/js/CldrSurveyVettingLoader.js

```diff
@@ -641,141 +641,146 @@ function showV() {
-
-							{
-								divider: true
-							},
-							{
-								title: 'My Organization(' + organizationName + ')'
-							}, // My Organization section
-
-							{
-								special: 'vsummary', /* Cf. special_vsummary */
-								level: 2,
-								display: surveyUserPerms.userCanUseVettingSummary
-							},
-							{
-								title: 'List ' + org + ' Users',
-								level: 2,
-								url: surveyUserURL.manageUser,
-								display: (surveyUserPerms.userIsTC || surveyUserPerms.userIsVetter)
-							},
-							{
-								special: 'forum_participation', /* Cf. special_forum_participation */
-								level: 2,
-								display: surveyUserPerms.userCanMonitorForum
-							},
-							{
-								title: 'LOCKED: Note: your account is currently locked.',
-								level: 2,
-								display: surveyUserPerms.userIsLocked,
-								bold: true
-							},
-
-							{
-								divider: true
-							},
-							{
-								title: 'Forum'
-							}, // Forum section
-
-							{
-								special: 'flagged',
-								level: 2,
-								img: surveyImgInfo.flag
-							},
-							{
-								special: 'mail',
-								level: 2,
-								display: !surveyOfficial
-							},
-							{
-								special: 'bulk_close_posts', /* Cf. special_bulk_close_posts */
-								level: 2,
-								display: (surveyUser && surveyUser.userlevelName === 'ADMIN')
-							},
-
-							{
-								divider: true
-							},
-							{
-								title: 'Informational'
-							}, // Informational section
-
-							{
-								special: 'statistics',
-								level: 2
-							},
-							{
-								title: 'About',
-								level: 2,
-								url: surveyUserURL.about
-							},
-							{
-								title: 'Lookup a code or xpath',
-								level: 2,
-								url: surveyUserURL.browse,
-								display: surveyUserPerms.hasDataSource
-							},
-
-							{
-								divider: true
-							},
+						/*
+						 * This indirectly references "special_oldvotes" in stui.js
+						 */
+						{
+							special: 'oldvotes',
+							level: 2,
+							display: surveyUserPerms.userCanImportOldVotes
+						},
+						{
+							title: 'See My Recent Activity',
+							level: 2,
+							url: surveyUserURL.recentActivity
+						},
+						{
+							title: 'Upload XML',
+							level: 2,
+							url: surveyUserURL.xmlUpload
+						},
+
+						{
+							divider: true
+						},
+						{
+							title: 'My Organization(' + organizationName + ')'
+						}, // My Organization section
+
+						{
+							special: 'vsummary', /* Cf. special_vsummary */
+							level: 2,
+							display: surveyUserPerms.userCanUseVettingSummary
+						},
+						{
+							title: 'List ' + org + ' Users', // 'org' from ajax_status.jsp
+							level: 2,
+							url: surveyUserURL.manageUser,
+							display: (surveyUserPerms.userIsTC || surveyUserPerms.userIsVetter)
+						},
+						{
+							special: 'forum_participation', /* Cf. special_forum_participation */
+							level: 2,
+							display: surveyUserPerms.userCanMonitorForum
+						},
+						{
```

Comment:
is this block for vetting_participation the only non-whitespace change in this function?

Reply:
I don't know what happened with the whitespace here. Can you hit the Gear and review with 'ignore whitespace'?